framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -180,36 +184,59 @@ export const CreatableSelectDropdown = <T extends string = string> ({
   ]
 
   return (
-    <Box ref={ ref }>
-      <CreatableSelect
-        menuPlacement={ menuPlacement }
-        chakraStyles={ {
-          ...customSelectStyles,
-          container: (provided) => ({
-            ...provided,
-            width,
-          }),
-          option: (provided, { isSelected }) => ({
-            ...provided,
-            ...(isSelected && {
-              color: 'black',
+    <Box ref={ ref } w="sm" maxW="full">
+      { createNewOption ? (
+        <EditableText
+          value={ addFieldInputValue }
+          startWithEditView={ true }
+          onChange={ (v) => {
+            setAddFieldInputValue(v)
+          } }
+          submitOnBlur={ false }
+          onCancel={ () => {
+            setAddFieldInputValue('')
+            toggleCreateNewOption()
+          } }
+          onSubmit={ (v) => {
+            handleCreateOption(v)
+          } }
+          variant="brand"
+          sx={ {
+            minWidth: '100%',
+            width: '100%',
+          } }
+        />
+      ) : (
```

Comment:
We want to avoid the ternary operator in jsx code. Love it otherwise but rendering components like makes it a bit unreadable.
So we prefer

```
{ condition && (
  <ComponentIfTrue />
) }
{ !condition && (
  <ComponentIfFalse />
) }
```